ansible/playbooks/main.yaml

```diff
@@ -0,0 +1,5 @@
+- hosts: molbertevm
+  roles:
+    - name: Install Docker
+      role: geerlingguy.docker
```

Comment:
nice ![](https://steamcommunity-a.akamaihd.net/economy/emoticon/:ohh_yeah:)
i now question the meaning of my life because i wrote that role from scratch by myself :trollface: